Gini-iOS-SDK/GINIDocument.h

```diff
@@ -53,21 +56,27 @@ typedef NS_ENUM(NSUInteger, GiniDocumentSourceClassification) {
 @property NSDate *creationDate;
 /// The document's source classification.
 @property GiniDocumentSourceClassification sourceClassification;
+/// Links to related resources, such as extractions, document, processed or layout.
+@property (readonly) GINIDocumentLinks *links;
+/// (Optional) Array containing the path of every parent
+@property (readonly) NSArray<NSString *> *parents;
+/// (Optional) Array containing the path of every partial document
+@property (readonly) NSArray<NSString *> *partialDocuments;
 /// A `BFTask*` resolving to a mapping with extractions (extraction name as key).
-@property (readonly) BFTask *extractions;
+@property (readonly) BFTask *extractions __attribute__((unavailable("use `GINIDocumentTaskManager.getExtractionsForDocument:` method instead")));
```

Comment:
Don't you have to deprecate it first?

Reply:
I was thinking to make them `unavailable` since it is a major version, but we can actually deprecate them now and remove them in a future version.